app/controllers/sdb/sample_manifests_controller.rb

```diff
@@ -55,8 +55,10 @@ def show
   end
 
   def index
-    pending_sample_manifests = SampleManifest.pending_manifests.paginate(page: params[:page])
-    completed_sample_manifests = SampleManifest.completed_manifests.paginate(page: params[:page])
+    pending_sample_manifests =
+      SampleManifest.pending_manifests.includes(:study, :supplier, :user).paginate(page: params[:page])
+    completed_sample_manifests =
+      SampleManifest.completed_manifests.includes(:study, :supplier, :user).paginate(page: params[:page])
```

Comment:
Performance optimisation. Not entirely convinced by the split, feel like it may make morse sense to allow filtering of manifests by state.